omnigibson/utils/asset_utils.py

```diff
@@ -11,26 +11,20 @@
 import omnigibson as og
 from omnigibson.utils.ui_utils import create_module_logger
 
-if os.name == "nt":
-    import win32api
-    import win32con
-
 # Create module logger
 log = create_module_logger(module_name=__name__)
 
 
 def folder_is_hidden(p):
     """
-    Removes hidden folders from a list. Works on Linux, Mac and Windows
+    Check if a folder is hidden. This corresponds to checking if its filename starts with a dot.
+    Note that while this does not actually correspond to checking for hidden files on Windows, the
```

Comment:
nit, maybe we should rename this function so it semantically matches this docstring? e.g.: `dot_folder`

Reply:
done